content/docs-v1/collection-catalogues.md

```diff
@@ -1,126 +1,323 @@
+object_list:
+  - id: cat-1
+    artist:
+    title:
+    year:
+    dimensions:
+    medium:
+    location:
+    type:
+    thumbnail: figures/lange-mother.jpg
+    figure: 
+      - id: fig-1-1
+      - id: fig-1-2
+      - id: fig-1-3
 ```
 
+## Create Object Pages
+
+Like all other pages in your publication, object pages are created as `.md` files in your `content` directory. To create an object entry page, add `layout: entry` and either the object data or the `id` that corresponds with your `objects.yaml` file to the page YAML. 
+
+**Object data in page YAML:**
+
+```yaml
+---
+title: 
+layout: entry
+order:
+object:
+  - artist:
+    title: 
+    year:
+    dimensions:
+    medium:
+    location:
+    type:
+    figure:
+      - id:
+---
+```
+
+**Object data that references `objects.yaml` file:**
+
+```yaml
+---
+title:
+layout: entry
+order:
+object:
+  - id: cat-1
+---
+```
+
+There are two different ways you can format your object entry pages. Adding `presentation: side-by-side` to the page YAML will make the figure image and tombstone appear next to each other and `presentation: landscape` will make the image appear above the tombstone information. You can set a default format for all your entry pages in the `content/_data/config.yaml` by choosing either `true` or `false` for the `sideBySide` attribute.
+
 {{< q-figure id="modern-entry-side" >}}
 
+{{< q-figure id="modern-entry" >}}
+
+{{< q-class "box tip" >}}
+- `presentation: side-by-side` layouts will automatically switch to landscape mode on smaller screens.
+{{< /q-class >}}
+
 Whatever the presentation, you can further organize the content on the page by using Quire’s `{% accordion %}` shortcode to wrap the individual parts of the object entry into collapsible sections. Read more in the [*Accordions*](/docs-v1/accordions/) chapter of this guide.
 
 {{< q-figure id="modern-entry-side-accordion" >}}
 
+## Generate Object Grids
 
-### Object Images
+In a collection catalogue, there is typically a visual table of contents that unifies all the catalogue entries. In Quire, we call this an object grid. The grid can include a thumbnail of each object accompanied by object data. Each item in the grid hyperlinks to the relevant object page in the catalogue when clicked. 
 
-You can add more than one image to your object entry page by listing multiple figure `ids` in your `objects.ymal` file. Multiple images are displayed as a rotating carousel and are arranged in the order they are listed in the `objects.yaml` file. If any of the object images have a `caption` and/or `credit`, that information will be included as a pop-up window.
+[Add screenshot - need advice]
 
-If the figure’s `download` attribute is set to `true`, a download icon will be included as well.
+To create an object grid, there are three key steps. Add `layout: object-grid` to a designated `.md` file, add an `object-card` list to your `objects.yaml` file, and add an `object-filters` list to your `objects.yaml` file.
```

Comment:
It's `object_card` and `object_filters` with underscores instead of hyphens. Check/change throughout